frontend/src/metabase/components/FieldValuesWidget/FieldValuesWidget.tsx

```diff
@@ -466,6 +499,20 @@ export function FieldValuesWidgetInner({
             optionRenderer={optionRenderer}
             checkedColor={checkedColor}
           />
+        ) : !isSimpleInput ? (
+          <MultiAutocomplete
+            onSearchChange={onInputChange}
+            onChange={onChange}
+            value={value.filter(v => v !== null && v !== undefined)}
+            data={options.map(renderStringOption)}
+            renderValue={value => optionRenderer?.([value])}
+            placeholder={tokenFieldPlaceholder}
+            shouldCreate={shouldCreate}
+            autoFocus={autoFocus}
+            prefix={prefix}
+            parseValue={parseFreeformValue}
```

Comment:
We shouldn't need to pass this to the component; it could be handled here, outside of the component, like `FilterValuePicker` does.